Source/WebKit/Shared/DisplayLinkObserverID.h

```diff
@@ -32,4 +32,7 @@ namespace WebKit {
 enum DisplayLinkObserverIDType { };
 using DisplayLinkObserverID = ObjectIdentifier<DisplayLinkObserverIDType>;
 
+enum DisplayLinkObserverCollectionIDType { };
+using DisplayLinkObserverCollectionID = ObjectIdentifier<DisplayLinkObserverCollectionIDType>;
```

Comment:
Where is enum used? I do not see it in this PR.

Reply:
Agree, this is leftover.